### DISCUSSION TITLE
How to establish one-to-one, one to many, and many to many relationships between ContentItems,please?

### DISCUSSION THREAD

**Discussion by effapp** [OrchardCMS/OrchardCore#15207] [Q&A] [answered]
Hi,
How to establish one-to-one, one to many, and many to many relationships between ContentItems,please?

**Comment by MikeAlhayek** [comment, marked as answer]
Please only use issue when there is an issue. Otherwise, please use discussion.

In OC there is no relations between table as you would see in typical ORM or database structure. 

You can use ContentPickerField on ContentTypeA so then the user create an instance of `A` they would have a way to select a single or multiple ContentTypeB.

**Comment by effapp** [comment]
Thanks.

**Reply by hishamco** [reply]
Also, you could use BagPart & Lists

**Comment by Piedone** [comment]
I sure wish a guy who is not only smart but very handsome too would've written a docs page about just this: https://docs.orchardcore.net/en/latest/docs/topics/content-relationships/ :D.

**Reply by hishamco** [reply]
Thanks handsome guy :)
